helpers/hunit_test.py

```diff
@@ -617,8 +617,8 @@ def purify_txt_from_client(txt: str) -> str:
     """
     Remove from a string all the information of a specific run.
     """
-    txt = purify_from_environment(txt)
     txt = purify_app_references(txt)
+    txt = purify_from_environment(txt)
```

Comment:
If there is an order we should explain why and how.

It's best to make all the calls idempotent to avoid these problems